views/hoc/withRoot.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { MuiThemeProvider } from '@material-ui/core/styles'
+import CssBaseline from '@material-ui/core/CssBaseline'
+
+function withRoot(Component) {
+  function WithRoot(props) {
+    return (
+      <MuiThemeProvider>
+        <CssBaseline />
```

Comment:
can we wrap the entire app in this and not need to wrap each individual component?